changelog/7652.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix the :property:`~sunpy.map.sources.gong.GONGHalphaMap.rsun_obs` to use correct header information ``solar-r`` keyword.
```

Comment:
Hum docs build doesn't seem to like `:property:` maybe this is the wrong syntax?